paddlets/analysis/time_domain.py

```diff
@@ -84,7 +84,7 @@ def analyze(
     def _seasonality(
         self, 
         X: pd.DataFrame,
-        period_dict: dict = {},
+        period_dict: Optional[dict] = {},
```

Comment:
should Not be Optional, refers to https://docs.python.org/3/library/typing.html#typing.Optional